### PR TITLE
Wr/npm pin

### PR DESCRIPTION
**Pull request by WillieRuemmele**

### CODE REVIEW

File: src/jobs/pin-dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            name: dependencies:pin
+            command: sf-release npm:dependencies:pin --tag <<parameters.tag>>
+        - run:
+            name: commit package.json change
+            command: |
+              git status;
+              # if we have ONLY changed the package.json then we can commit and push
+              if  git status --porcelain v1 | grep 'package.json' && git status --porcelain v1 | wc -l == 1; then
+                git add package.json
+                git commit -m "Automatically adding package.json [skip ci]";
```
Comment (mdonnalley) [inline src/jobs/pin-dependencies.yml:43]: I think we should change this to something like this:

```
"chore: pin dependencies in package.json [skip ci]"
```

**Review by mdonnalley — CHANGES_REQUESTED** [review]

File: src/jobs/pin-dependencies.yml (hunk 1 above)
Comment (WillieRuemmele) [reply src/jobs/pin-dependencies.yml:43]: I agree

**Review by mdonnalley — APPROVED** [review]

**Comment by WillieRuemmele** [comment]
[semver:patch] your commit message

**Review by shetzel — APPROVED** [review]
lgtm

**Pull request merged**